### PR TITLE
EG-4165 Correct license for New Relic

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

**Review by ivanterziev-r3 — DISMISSED** [review]
LGTM

**Review by AMcFarlandCampbellR3 — APPROVED** [review]

File: content/en/docs/corda-os/4.3/legal-info.md [hunk 1]
```diff
@@ -708,6 +708,10 @@ snakeyaml 1.18
 [http://www.snakeyaml.org/](http://www.snakeyaml.org/)
 [https://bitbucket.org/asomov/snakeyaml/src/default/LICENSE.txt](https://bitbucket.org/asomov/snakeyaml/src/default/LICENSE.txt)
 
+newrelic-api    3.10.0
+[https://newrelic.com/](https://newrelic.com/)
+[https://github.com/newrelic/newrelic-java-agent/blob/main/LICENSE)
+
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.3/legal-info.md:714]: mnesbit 1 hour ago 
Our code in node build.gradle only links against com.palominolabs.metrics:metrics-new-relic:${metrics_new_relic_version} not the agent? The user has to provide the agent?

Comment (rnicoll) [reply content/en/docs/corda-os/4.3/legal-info.md]: Gradle and Nexus IQ both suggest we're shipping the agent as a transitive dependency of `com.palominolabs.metrics:metrics-new-relic`, but if it shouldn't be packaged in we can eliminate that dependency

**Comment by ivanterziev-r3** [comment]
Closing and will open a clean new PR following the latest changes as per #299

**Pull request closed without merging**